web/themes/custom/usagov/scripts/modal.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * This script displays a modal upon page load if the modal is present on the page and
+ * the URL includes the modal's name as the 'modal' query parameter.
+ *
+ * The toggleModal, openModal, and closeModal functions can be used to control modals
+ * from other scripts. These functions take a modal ID argument which is a numerical ID
+ * rather than the modal's name, so we rely on the modalID function to get the ID for a
+ * given modal name.
+ */
+
+document.addEventListener('DOMContentLoaded', function() {
+    "use strict";
+
+    document.querySelectorAll('.usa-modal').forEach((modal) => {
+        // Modals start hidden to prevent a flash of content on page load.
+        // This unhides the modals after USWDS script has moved them into a hidden container.
+        modal.hidden = false;
+    });
+
+    let modalName = removeUrlParameter('modal');
```

Comment:
Let's sanitize this modalName here. (I think here is better than within "removeUrlParameter." )